server/elm/src/Pages/Scoreboard/View.elm

```diff
@@ -1,23 +1,536 @@
 module Pages.Scoreboard.View exposing (view)
 
 import App.Types exposing (Language)
+import AssocList as Dict exposing (Dict)
+import Backend.Entities exposing (fromEntityId, toEntityId)
+import Backend.Model exposing (ModelBackend)
+import Backend.Scoreboard.Model exposing (ScoreboardData, SelectedEntity(..))
+import Date
+import Gizra.Html exposing (emptyNode, showIf)
+import Gizra.NominalDate exposing (NominalDate)
 import Html exposing (..)
-import Html.Attributes
-    exposing
-        ( class
-        , classList
-        , href
-        , id
-        , src
-        , style
-        )
-import Html.Events exposing (onClick)
+import Html.Attributes exposing (..)
 import Icons
 import Maybe.Extra exposing (isJust, isNothing)
-import Pages.Scoreboard.Model exposing (Model, Msg(..))
+import Pages.Scoreboard.Model exposing (..)
 import Pages.Scoreboard.Utils exposing (..)
+import Pages.Utils exposing (viewYearSelector)
+import Time exposing (Month(..))
+import Translate exposing (TranslationId, translate)
+import Utils.GeoLocation exposing (GeoLocationId, filterGeoLocationDictByParent, geoInfo, geoLocationDictToOptions)
+
+
+view : Language -> NominalDate -> ModelBackend -> Model -> Html Msg
+view language currentDate modelBackend model =
+    case modelBackend.scoreboardData of
+        Just (Ok data) ->
+            viewScoreboardData language currentDate data model
+
+        Just (Err err) ->
+            text <| Debug.toString err
+
+        Nothing ->
+            emptyNode
+
+
+viewScoreboardData : Language -> NominalDate -> ScoreboardData -> Model -> Html Msg
+viewScoreboardData language currentDate data model =
+    let
+        topBar =
+            div [ class "top-bar" ]
+                [ div [ class "new-selection" ]
+                    [ a [ href "/admin/reports/aggregated-ncda" ]
```

Comment:
Where do we check if this given user has access to that URL? Or is it accessible for all users?

Reply:
Yep, but can it happen that we display this link, but they get a 403?

Reply:
But it's good that you mentioned it, as I now see that a bug I fixed at https://github.com/TIP-Global-Health/eheza-app/pull/706/commits/c0b1e4280116555448a71973577937eec7cb3d36 reapeared again.

Fixed it at [06003d1](https://github.com/TIP-Global-Health/eheza-app/pull/706/commits/06003d17dc6cc8dc06f551864fbd9cbc5662cf37).

Reply:
Well, all pages got same access check. So, if you're on page that displays the link, you can access the link.